### PR TITLE
Story endpoint integration

### PR DESCRIPTION
**Pull request by bigardone**
Hi!
Here's the **Story** endpoint implementation for finding a **Story** by `project_id` an `story_id`.
Cheers!

### CODE REVIEW

File: lib/extracker/stories.ex [hunk 1]
```diff
@@ -3,6 +3,22 @@ defmodule ExTracker.Stories do
   alias Extracker.Client
 
   @doc """
+  Get a single `story`.
+
+  ## Example
+
+      ExTracker.Stories.find("12345", "67890", client)
+
+  More info at: https://www.pivotaltracker.com/help/api/rest/v5#Story
```
Comment (forest) [inline lib/extracker/stories.ex]: @bigardone what do you think about using the endpoint that just requires the `story_id` and not the `project_id`?
https://www.pivotaltracker.com/help/api/rest/v5#stories_story_id_get

The `project_id` isn't required to get a story, and it allows a bit more flexibility for the user.

Comment (bigardone) [inline lib/extracker/stories.ex]: You're right, let me change it :)

**Comment by bigardone** [comment]
Changes done :)
There's a minor detail I wanted to comment with you. In some cases we pass the client as first argument and in other cases we pass it as the last one. While using the library is a bit confusing remembering each case... do you mind if I do some refactoring so the client gets always passed as the last argument everywhere?

**Comment by forest** [comment]
@bigardone: I agree inconsistency is bad.

I think it would be better to make `client` the first param in all cases for two reasons.
1. It is sometimes the only required param.
    `Projects.list(client)`
2. It allows for pipelining.
    `Projects.find(client, "123456")`
    `client |> Projects.find("123456")`

```
stories = with client <- @client,
               [project | _] <- Projects.list(client),
            do: Stories.list(client, project.id)
```

**Comment by bigardone** [comment]
Makes sense, I'll refactor everything then so it's passed as the first argument :)

**Comment by forest** [comment]
:+1:

**Pull request merged**